GameOfLife/GameOfLife/Field.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    public class Field
+    {
+        private int _fieldHeight { get; set; }
+        private int _fieldWidth { get; set; }
+
+        private string[,] fieldArray;
+
+        public Field(int height, int width)
+        {
+            _fieldHeight = height;
+            _fieldWidth = width;
+        }
+
+        /// <summary>
+        /// Initial creation of an empty gaming field.
+        /// </summary>
+        /// <returns></returns>
+        public string[,] CreateField()
+        {
+            fieldArray = new string[_fieldWidth, _fieldHeight];
+
+            for (int j = 0; j < _fieldHeight; j++)
+            {
+                for (int i = 0; i < _fieldWidth; i++)
+                {
+                    fieldArray[i, j] = "-";
+                }
+            }
+
+            return fieldArray;
+        }
+
+        /// <summary>
+        /// Function that draws the field.
+        /// </summary>
+        /// <param name="field"></param>
+        public void DrawField(string [,] field)
+        {
+            Console.WriteLine();
+
+            for (int j = 0; j < field.GetLength(0); j++)
+            {
+                for (int i = 0; i < field.GetLength(1); i++)
+                {
+                    Console.Write(" " + field[i, j]);
+                }
+
+                Console.WriteLine();
+            }
+        }
+
+        /// <summary>
+        /// Seeding the field with user input cells.
+        /// </summary>
+        /// <returns></returns>
+        public string[,] SeedField()
```

Comment:
This method looks too long. Split it in two separate methods. One to generate data manually other to generate data at random. Inquiry about which option to use should be put in other method, maybe master method.

Reply:
Split into two methods - ManualSeeding() and RandomSeeding(), which are called using ChooseSeedingOption() method.